sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusServiceVersion.java

```diff
@@ -8,7 +8,8 @@
  * The versions of Azure Service Bus supported by this client library.
  */
 public enum ServiceBusServiceVersion implements ServiceVersion {
-    V2017_04("2017-04");
+    V2017_04("2017-04"),
```

Comment:
When the customer targets this older API version, the maxMessageSizeInKilobytes property will be null in the response from the service. JS & .NET had to change this property to be nullable to support this case. Would you need a similar approach in Java?

Reply:
this is not required property in Java, so we should be good here